src/redux/publishers/publishers.js

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from "@reduxjs/toolkit";
+import axios from "axios";
+import url from "../../api";
+
+const initialState = {
+	publishers: [],
+};
+
+const publishers = createSlice({
+	name: "publishers",
+	initialState,
+	reducers: {
+		getPublishers: (state, action) => {
+			state.publishers = action.payload;
+		},
+	},
+});
+
+export const { getPublishers } = publishers.actions;
+
+export const fetchPublishers = () => async dispatch => {
```

Comment:
because you are not using the parameter of this function you can remove those brackets like this
```suggestion
export const fetchPublishers = async dispatch => {
```